season/week_two_challenges/more_methods.md

```diff
@@ -0,0 +1,117 @@
+# More Method Madness
+
+## Method of Thrones
+
+Create a method called `character_house`.
+* This method should take two string arguments: `first_name` and `house_name`.
+* When executed, the method should return "`first_name`, house of `house_name`".
+
+Example:
+```ruby
+character_house("Tyrion", "Lannister")
+# output => "Tyrion, house of Lannister."
+```
+```ruby
+def character_house(first_name,house_name)
```

Comment:
Nitpick: put a space between arguments (e.g. `def character_house(first_name, last_name)`. You did this in a few spots, so check everywhere.